### ISSUE TITLE
[MU4 Issue] Engraving of two-note tremolos is not ideal

### ISSUE THREAD

**Issue by Harmoniker1** [musescore/MuseScore#15470]
**Describe the bug**
It is reported in [this issue thread](https://musescore.org/en/node/339387) that the layout of two-note tremolos is not as good as in 3.6.
* In 3.6, the two chords that share a two-note tremolo can have different stem directions, which is not allowed in 4.0.
* The layout in 3.6 is also better when they have the same stem direction.

See these images for comparison:
![image](https://user-images.githubusercontent.com/46259489/208883153-463be7eb-743b-4cc6-8f84-ee6e7c657eb6.png) (MuseScore 3.6)
![image](https://user-images.githubusercontent.com/46259489/208883473-aa733978-0520-4108-a43b-3530454d7374.png) (MuseScore 4.0 - the other one is not possible to achieve)

**Platform information**
 - OS: Windows 11

**Comment by SteveBlower** [comment]
Also reported on .org Issue Tracker https://musescore.org/en/node/339387

**Comment by oktophonie** [comment]
In an ideal world, tremolos would have been implemented as a flavour of beam, and so all the new beam placement code would apply to them to. Sadly this didn't turn out to be the case (they were a thing all unto themselves), and changing this required a refactoring effort that we didn't have time for for 4.0, so their current placement is, sadly, pretty gruesome.

Rest assured that this is one of my highest prioirities for 4.1, though.

**Comment by Harmoniker1** [comment]
@oktophonie That's glad to hear! I would totally support treating two-note tremolos like beams. But meanwhile, does the current "gruesomeness" deserve a quick patch fix for minor releases?

**Comment by oktophonie** [comment]
Not really interested in quick hacks to cover up larger problems; that's precisely what we're trying to get away from.

**Comment by Tantacrul** [comment]
> Not really interested in quick hacks to cover up larger problems; that's precisely what we're trying to get away from.

Exactly. When we do it right, we can then build upon that work. A hack has to be eventually removed, which then messes up people's scores.
